templates/userservice/userservice-deployment.yaml

```diff
@@ -491,6 +491,18 @@ spec:
                 configMapKeyRef:
                   key: SPRING_SLEUTH_SAMPLER_PERCENTAGE
                   name: userservice-{{ .Release.Name }}-env
+          {{ - if .Values.global.site24x7Enabled }}
```

Comment:
please test if this is working for other instances like digi where no valeues.global.site24xEnabled is defined

Reply:
we need to make sure that we're not breaking exisiting pipelines. Probably we'll need to set default value to false here.